integration/browser-script.js

```diff
@@ -0,0 +1,30 @@
+import { browser } from 'k6/browser';
+import { check } from 'https://jslib.k6.io/k6-utils/1.5.0/index.js';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
+    checks: ['rate==1.0'],
+  },
+};
+
+export default async function () {
+  const context = await browser.newContext();
+  const page = await context.newPage();
+
+  try {
+    // e-commerce site as a torture test for metric generation.
+    await page.goto('https://www.amazon.com');
```

Comment:
Rather than depending on an external source, we could use custom metrics in k6 but with the builtin names. This would likely increase the stability but it a little tedious. An idea for the future maybe!

Reply:
Yep, I think producing a torture-test locally would be a good next step!
Ideally we should return some actual HTML with loads of (mocked) images and scripts.